src/utils.ts

```diff
@@ -4,6 +4,10 @@ export function encode(object: Record<string, unknown>): string {
   return JSON.stringify(object);
 }
 
+export function encodeArray(object: Array<Record<string, unknown>>): string {
```

Comment:
We should be able to use `encode` instead of making a new function. The type signature for `encode` would need to be updated to something like `Record<string, unknown> | Array` I think.

Reply:
Oh if that works, it would be great. As both of these do exactly same thing. I just figured below works as you mentioned:
```ts
export function encode(
  object: Record<string, unknown> | Array<Record<string, unknown>>
): string {
  return JSON.stringify(object);
}
```

Lack of experience on Typescript :)